lib/services/execution.dart

```diff
@@ -21,6 +21,8 @@ abstract class ExecutionService {
     String css,
     String javaScript, {
     String /*?*/ modulesBaseUrl,
+    bool addRequireJs = false,
```

Comment:
nit: we might want to add a comment that addRequireJs should be true when executing DDC javascript

Reply:
Good catch. I've added documentation to this method.